share/completions/sv.fish

```diff
@@ -0,0 +1,80 @@
+# Completions for sv
+# A part of the runit init system
+# Author: Leonardo da Rosa Eugênio <lelgenio@disroot.org>
+
+
+set -l commands \
+    status up down once s u d o pause cont hup \
+    alarm interrupt quit 1 2 term kill exit p c h \
+    a i q 1 2 t k e start stop reload restart \
+    shutdown force-stop force-reload force-restart force-shutdown \
+    try-restart check
+
+
+function __fish_complete_sv_list_services
+    for dir in "$SVDIR" /run/runit/service/ /etc/runit/current /etc/runit/runsvdir/
+        test -d "$dir"
+        and break
+    end
+    set -l services (command ls "$dir")
+    if sv status "$services[1]" &> /dev/null
+        for service in $services
+            printf "%s\t%s\n" "$service" (sv status "$service")
+        end
+    else
+        printf "%s\n" $services
+    end
+end
+
+complete -f -c sv -a "(__fish_complete_sv_list_services)" -n "__fish_seen_subcommand_from $commands"
+
+
+complete -fc sv -s v -d "Report status for up, down, term, once, cont, and exit"
+complete -fc sv -s w -d "Override the default timeout to report status"
+
+
+set -l no_comm "not __fish_seen_subcommand_from $commands"
+
+complete -kfc sv -n "$no_comm" -a check           -d "Check if the service is in it's requested state"
+complete -kfc sv -n "$no_comm" -a try-restart     -d "Run term, cont, and  up, report status"
+complete -kfc sv -n "$no_comm" -a force-shutdown  -d "Run exit, report status or kill on timeout"
+complete -kfc sv -n "$no_comm" -a force-restart   -d "Run term, cont and up, report status"
+complete -kfc sv -n "$no_comm" -a force-reload    -d "Run term and  cont, report status"
+complete -kfc sv -n "$no_comm" -a force-stop      -d "Run down, report status or kill on timeout"
+complete -kfc sv -n "$no_comm" -a shutdown        -d "Run exit, report status"
+complete -kfc sv -n "$no_comm" -a restart         -d "Run term, cont, and  up, report status using ./check"
+complete -kfc sv -n "$no_comm" -a reload          -d "Run hup, report status"
+complete -kfc sv -n "$no_comm" -a stop            -d "Run down, report status"
+complete -kfc sv -n "$no_comm" -a start           -d "Run up, report status"
+
+complete -kfc sv -n "$no_comm" -a e         -d "Alias for exit"
+complete -kfc sv -n "$no_comm" -a k         -d "Alias for kill"
```

Comment:
although it looks nice, we usually don't align arguments - care to run `fish_indent` on this file?